Sources/BraveNews/FeedCardContent.swift

```diff
@@ -37,4 +37,8 @@ public enum FeedItemAction: Equatable {
   case opened(inNewTab: Bool = false, switchingToPrivateMode: Bool = false)
   /// Toggle the enabled status for the source of the feed item
   case toggledSource
+  /// The user choose to navigate Appstore for app rating
+  case pressedAppRatingCard
+  /// The user choose to hide card for app rating
+  case hiddenAppRatingCard
```

Comment:
Lets remove these from here and make the `actionHandler` just an unused variable on the ratings card. We can have a separate action setup for the ratings card instead

Reply:
i.e. 

```swift
enum RatingCardAction {
  case rateBrave
  case hideCard
}
```

Reply:
Created a new enum at https://github.com/brave/brave-ios/pull/7589/commits/a4f446ca688845f1e0b47127a94c5b28f4e09231